pkgs/development/libraries/hyperscan/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchurl
+, pkgconfig
+, boost
+, cmake
+, python
+, ragel
+}:
+
+let
+
+in
+stdenv.mkDerivation rec {
+  name = "hyperscan-${version}";
+  version = "4.6.0";
+
+  src = fetchurl {
+    url = "https://github.com/intel/hyperscan/archive/v${version}.tar.gz";
+    sha256 = "05d8zpjcm13lldkzcxafkabllzgv10nrhgfazsrag9l2bqpgryqd";
+  };
+
+  nativeBuildInputs = [
+    boost
+    cmake
+    pkgconfig
+    python
```

Comment:
Are `boost`, `python` and `ragel` build-time only dependencies in this case? Perhaps they are, I just wanted to make sure.

Reply:
Yes.